### ISSUE TITLE
Switch to JsonField for the additional data in the ProcessedCrash

### ISSUE THREAD

**Issue by mmohrhard** [mmohrhard/crash#23]

**Comment by mmohrhard** [comment]
Currently the additional data in each processed crash is stored as json strings. The used postgresql database provides a better solution for storing json through the JsonField model class. This would allow to write better statistic queries.

**Comment by self-maurya** [comment]
@mmohrhard 
are you referring to [this](https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/fields/#jsonfield) ?
I want to help if this is still up for grab.

**Comment by mmohrhard** [comment]
Yes. I'm talking about the JsonField from the postgresql models. The hope is that it will make it easier to create statistics across the crash report information that are stored in the additional data.

**Comment by duilio92** [comment]
Hi, I guess this is still up for grabs since two years passed, will be working on this if that's okey.
